specification/compliance.md

```diff
@@ -0,0 +1,19 @@
+# FOCUS Compliance
+
+Most billing datasets are unique based on the provider, its method of determining cost (e.g. fixed vs consumption-based billing), support for various concepts (e.g. commitment-based discounts), and the provider's billing system's maturity level.
+
+FOCUS aims to help FinOps practitioners perform common FinOps capabilities by using a generic set of instructions. FOCUS accomplishes this by defining compliance requirements that account for the provider-level differences mentioned above.
+
+## Enabling FinOps Capabilities For Heterogeneous Cost Datasets
+
+Cloud Service Providers (CSPs) generate detailed cost and usage data in near real-time enabling an extensive set of FinOps capabilities to drive accountability and efficiency for dynamic consumption-based costs.  Non-CSP providers typically provide billing datasets that lack the same level of detail. Primary FinOps capabilities like invoice reconciliation, data ingestion, and data normalization can be performed even on less-detailed billing datasets if an essential set of requirements are met. Simplified versions of additional FinOps capabilities like cost reporting, chargeback, showback, and budgeting/forecasting can also be performed using a less-detailed billing dataset when essential requirements are met.
+
+Performing these capabilities will ultimately benefit organizations by enabling efficient, data-driven decision-making. The essential requirements needed to perform the important FinOps capabilities defined above are hereby referred to as the 'FOCUS Essential' requirements.
+
+## Meeting FOCUS Requirements
+
+FOCUS [Columns](#columns) designated as FOCUS Essential are required to be present in all compliant datasets and comply with the requirements (e.g. nullability and various other semantics) specified in the normative prose of such columns.
```

Comment:
Non-essential columns should also comply with the requirements, if they are present, shouldn't they?

Reply:
oops i may have made my change as a commit rather than a comment @udam-f2 LMK if you want me to rollback??